grails-app/conf/Config.groovy

```diff
@@ -128,7 +128,7 @@ openId {
 }
 
 featureToggles {
-    // someFeature = <boolean>
+    filterSource = 'database' // Todo - DN: This can go back to being a boolean when we've removed loading filters from the DB
```

Comment:
Correct, no_db should amend this to become a boolean rather than a string. Isn't that what you intended to do?

I mean, it can be factored to something like `dynamicFilters = true/false`. And then have (not as a toggle) `filterSource = 'database/files`, and `filterSource` would actually be used only if dyanmic filters is set to false - then you need "static" filters, either from database or files.

Either way, not a biggie. Even though @jkburges is concerned, I'm not very, because I know it'll be gone sooner than later.